freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/kerberosmgmt/doc/KeytabModelNotes.java

```diff
@@ -7,8 +7,9 @@
             + "the response is base64 encoded.";
     public static final String GET_KEYTAB_NOTES = "Retrieves the existing keytab for the service principal derived from the host and service provided. "
             + "Gets the existing keytab without modification not effeting the prior keytab. The keytab in the response is base64 encoded.";
```

Comment:
typo: "without modification not effeting" -> "without modification and not effecting" (I know you didn't change this for this code review)